### PR TITLE
(grc_conversion) EpiDoc and CTS conversion

### PR DESCRIPTION
**Pull request by lcerrato**
tlg0627 001-003 first conversion batch #1399

### CODE REVIEW

**Comment by lcerrato** [comment]
@AlisonBabeu Note the source (for applicable files) is the 1961 printing. Can be changed if needed.

File: data/tlg0627/tlg002/tlg0627.tlg002.perseus-grc2.xml
Comment (AlisonBabeu) [inline data/tlg0627/tlg002/tlg0627.tlg002.perseus-grc2.xml:1]: The title needs a "grc" tag `<title>Περὶ ἀέρων, ὑδάτων, τόπων</title>`

Comment (lcerrato) [reply data/tlg0627/tlg002/tlg0627.tlg002.perseus-grc2.xml:1]: Good catch!

File: data/tlg0627/tlg005/tlg0627.tlg005.perseus-grc2.xml
Comment (AlisonBabeu) [inline data/tlg0627/tlg005/tlg0627.tlg005.perseus-grc2.xml:1]: Just need to fix the file name, this is in the file: `<idno type="filename">tlg0627.tlg004.perseus-grc2.xml</idno>`

**Review by AlisonBabeu — CHANGES_REQUESTED** [review]
hi @lcerrato just two minor changes,looks good.

**Comment by lcerrato** [comment]
@AlisonBabeu 
Thank you, I didn't want to keep adding to this one.

**Comment by lcerrato** [comment]
> hi @lcerrato just two minor changes,looks good.

Should be incorporated now.

**Review by AlisonBabeu — APPROVED** [review]

**Pull request merged**